paddle/function/nnpack/NNPACKConvOp.cpp

```diff
@@ -58,18 +58,10 @@ class NNPACKConvFunction : public ConvFunctionBase {
     workspaceBuffer_ = nullptr;
     workspaceSize_ = 0;
 
-    threadpool_ = nullptr;
-    if (FLAGS_nnpack_num_threads) {
-      threadpool_ = pthreadpool_create(FLAGS_nnpack_num_threads);
-      VLOG(3) << "Number of threads "
-              << pthreadpool_get_threads_count(threadpool_);
-    }
+    create_nnpack_threadpool();
```

Comment:
In this way, we cannot destroy the `threadpool_` explicitly, and leave the work to OS?